packages/docs/src/locales/en/en.json

```diff
@@ -1138,11 +1138,16 @@
         "clearable": "Adds a button to reset the input field value",
```

Comment:
One small thing I noticed is that input resizes when you have warning/success icon.

![image](https://user-images.githubusercontent.com/5394573/148401100-84f2c480-21fe-4254-964c-e62fee941c19.png)

Reply:
Yes. It works fine if we need to show one icon, but slightly resize if there are more icons.

I made separated issue https://github.com/epicmaxco/vuestic-ui/issues/1411